cpp/CMakeLists.txt

```diff
@@ -16,10 +16,13 @@
 
 cmake_minimum_required (VERSION 3.11)
 
+project (libphonenumber VERSION 8.12.0)
```

Comment:
Could you please change the version to 8.13.0?

Reply:
Fixed with [3ce80e8](https://github.com/google/libphonenumber/pull/2818/commits/3ce80e80368daaadf3b315e2d4398eb52c7b7978).

Reply:
By the way, I removed calls to `set(libphonenumber_VERSION_MAJOR 8)` and co. They are unnecessary; CMake automatically sets the variables on call to `project(libphonenumber VERSION 8.13.0)`.